hack/images/README.md

```diff
@@ -13,5 +13,6 @@ The structure of the folder looks as follows:
 ```
   ├── jinja2                 # Image containing a containerized jinja CLI tool assisting with rendering templatized OCF content
   ├── json-go-gen            # The image that contains quicktype to generates Go struct from JSON Schemas
-  └── graphql-schema-linter  # The image that contains graphql-schema-linter to lint GraphQL files
+  ├── graphql-schema-linter  # The image that contains graphql-schema-linter to lint GraphQL files
+  └── merger                 # The image that contains yaml merger helper scriptyaml merger helper script
```

Comment:
```suggestion
  └── merger                 # The image that contains YAML merger helper
```
hm.. sth was merged here 🤔 :stuck_out_tongue_winking_eye:

Reply:
From what I see, during `argo retry` the pod is recreated(with the same name) and artifacts are downloaded again. We should be good here